src/library_webgpu.js

```diff
@@ -26,6 +26,7 @@
       return s;
     },
 
+    // TODO is getbool correct? It might not be if bools are packed as 8 bits
```

Comment:
oops, good point. sizeof(bool) is indeed 1. We can probably rely on that, but even if it were 4 bytes, it looks like we can read the address as i8 and it should work regardless of which way it is stored.

Reply:
The issue is that here we could be loading too much data, which would make it compare != 0 when in fact the bool itself is 0 but some neighbouring data is not.

Reply:
yeah, was proposing loading i8 instead. I already pushed a fix that does that